.github/workflows/ci.yaml

```diff
@@ -19,6 +19,7 @@ jobs:
       - uses: actions/checkout@v4
         with:
           repository: fireproof-storage/fireproof
+          ref: series_0.19
```

Comment:
I'll merge both and then we can remove this soon. since this passes with the 19 core, we can integration test this (manual) and merge to 019 / release it as part of that series

Reply:
@mabels the core repo's main branch is 0.20 now, but this repo's main branch is still 0.19... We'll merge this to the series 019 branch once we run acceptance integration tests on it